tensorflow/python/ops/control_flow_ops_test.py

```diff
@@ -936,7 +936,14 @@ def testCase_withoutDefault_oneCondition(self):
       with self.assertRaisesRegexp(errors.InvalidArgumentError, "Input error:"):
         sess.run(output, feed_dict={x: 4})
 
-
+  @test_util.run_in_graph_and_eager_modes
+  def testCase_dict_eagerMode(self):
```

Comment:
Nit: remove _eagerMode from the name of this test.

Reply:
Removed _eagerMode from the name